awsms-poc-alb.yml

```diff
@@ -45,6 +50,7 @@ Resources:
   APITargetGroup:

```

Comment:
Improvement:   For future we need to do this over HTTPS so that we are able to use e2e Encryption